src/main/java/frc/robot/subsystems/shooter/commands/bits/TestSolenoidPressure.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.subsystems.shooter.commands.bits;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.shooter.Shooter;
+
+import java.util.function.DoubleSupplier;
+
+import static frc.robot.Constants.Shooter.Bits.OUTPUT_MULTIPLIER;
+
+public class TestSolenoidPressure extends CommandBase {
```

Comment:
What solenoid?

Reply:
This year we're using a solenoid to change the angle of the shooter.